### ISSUE TITLE
Crash in SKTSRWebSocket

### ISSUE THREAD

**Issue by gemmakbarlow** [smooch/smooch-ios#107]
Full stack trace we just received via Fabric - only one instance in ~ 3k sessions, so not critical, but worth looking into I'd say. 

```
Crashed: Thread
0  libsystem_kernel.dylib         0x180ef411c __pthread_kill + 8
1  libsystem_pthread.dylib        0x180fc0ef8 pthread_kill + 112
2  libsystem_c.dylib              0x180e65dc8 abort + 140
3  libsystem_c.dylib              0x180e3ae7c basename + 322
4  Smooch                         0x1009c0684 -[SKTSRWebSocket dealloc] (SKTSRWebSocket.m:350)
5  Smooch                         0x1009c4fac -[SKTSRWebSocket _pumpScanner] (SKTSRWebSocket.m:1326)
6  Smooch                         0x1009c57d8 __37-[SKTSRWebSocket stream:handleEvent:]_block_invoke.468 (SKTSRWebSocket.m:1475)
7  libdispatch.dylib              0x180da54bc _dispatch_call_block_and_release + 24
8  libdispatch.dylib              0x180da547c _dispatch_client_callout + 16
9  libdispatch.dylib              0x180db14c0 _dispatch_queue_drain + 864
10 libdispatch.dylib              0x180da8f80 _dispatch_queue_invoke + 464
11 libdispatch.dylib              0x180da547c _dispatch_client_callout + 16
12 libdispatch.dylib              0x180db3914 _dispatch_root_queue_drain + 2140
13 libdispatch.dylib              0x180db30b0 _dispatch_worker_thread3 + 112
14 libsystem_pthread.dylib        0x180fbd470 _pthread_wqthread + 1092
15 libsystem_pthread.dylib        0x180fbd020 start_wqthread + 4
```

Our Podfile looks like

```
  pod 'Smooch', '4.2.2'          
  pod 'SmoochHelpKit', '1.1.0'    
```

**Comment by gemmakbarlow** [comment]
Occurred on an iPhone 6S running 9.3.3, plenty of space, App was in focus.

**Comment by mspensieri** [comment]
Fixed a number of crashes in [v6.1.0](https://github.com/smooch/smooch-ios/releases/tag/6.1.0). Please reopen if this one persists
